src/Symfony/Bridge/Monolog/Handler/DebugHandler.php

```diff
@@ -22,6 +22,29 @@
  */
 class DebugHandler extends TestHandler implements DebugLoggerInterface
 {
+    private $skip = true;
+
+    public function __construct($level = Logger::DEBUG, $bubble = true)
+    {
+        parent::__construct($level, $bubble);
+
+        if (func_num_args() >= 3) {
```

Comment:
I am not sure that we need to expose this option. It's only here to be able to run the tests for the class in the CLI environment.